guides/office/kitchen.md

```diff
@@ -9,9 +9,31 @@ We love our coffee at Made Tech. We have fresh beans regularly delivered to the
  - When grinding beans that no meetings are going on in the meeting room as it's very loud and can interrupt voice calls and conversations
  - When you're done making coffee that you tidy up any spillages so the kitchen is left in a clean state
 
-## Sink regime
+## Dirty glasses, cutlery, and crockery regime
 
-We all appreciate a clear and useable sink. Please ensure:
+We all appreciate a clean kitchen, with no dirty tableware, and a clear sink. This is why everyone should put their dirty tableware in the dishwasher.
 
- - When you're finished with glasses, cutlery and crockery that you wash them up
- - If you can't wash up for whatever reason you, rinse your glasses, cutlery and crockery and stack them to the side of the sink rather than in the sink, this is so that everyone can continue to use the sink throughout the day
+Before using the dishwasher, please familiarise yourself with [accepted dishwasher loading conventions](https://www.wikihow.life/Load-a-Dishwasher).
+
+Please ensure:
+
+ - When you're finished with any tableware that you rinse them and put them in the dishwasher
+ - When the dishwasher is full, then load it with a cleaning agent and start it
+ - When the dishwasher is running already, then wash your items in the sink with detergent instead
+ - When the dishwasher has finished and has clean items in it, empty it. Then put your dirty items inside!
+
+ Note: Please do not put your horse in the dishwasher.
+
+## Rubbish rules
+
+We are passionate about recycling at Made Tech, please familiarise yourself with which items should be recycled on [Southwark Council's](https://www.southwark.gov.uk/bins-and-recycling/recycling/what-goes-in-recycling) website.
```

Comment:
As we're a commercial premise, Southwark Council aren't responsible for our waste collection. See "What is Mixed Recycling / Dry Mixed Recycling?" here: https://thefirstmile.co.uk/contact#faqs (unfortunately you can't direct link to the specific FAQ :-()

Reply:
Ahhh! I'll have a look at this and update accordingly.